java/runtime/src/main/java/io/ray/runtime/AbstractRayRuntime.java

```diff
@@ -90,21 +95,35 @@ public AbstractRayRuntime(RayConfig rayConfig) {
       objectIds.add(objectRefImpl.getId());
       objectType = objectRefImpl.getType();
     }
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug("Getting Objects {}.", Arrays.toString(objectIds.toArray()));
```

Comment:
It't not necessary to call `objectIds.toArray()`, just use `List#toString`. If so, `LOGGER.isDebugEnabled()` is not necessary too.

Reply:
You mean `LOGGER.debug("Getting Objects {}.", objectIds);` is enough?